src/modules/sv_detection_methods/analyze_sa_tag_method.cpp

```diff
@@ -52,27 +52,27 @@ void retrieve_aligned_segments(std::string sa_string, std::vector<AlignedSegment
 
 void analyze_aligned_segments(const std::vector<AlignedSegment> & aligned_segments,
                               std::vector<Junction> & junctions,
+                              const seqan3::dna5_vector & query_sequence,
                               const std::string & read_name)
 {
     for(size_t i = 1; i<aligned_segments.size(); i++)
     {
         AlignedSegment current = aligned_segments[i-1];
         AlignedSegment next = aligned_segments[i];
         int32_t distance_on_read = next.get_query_start() - current.get_query_end();
-        // Neither gap nor overlap on read
-        if (distance_on_read >= -10 && distance_on_read <= 10)
+        // Check that there is neither an overlap nor a larger gap on the read
+        if (distance_on_read >= 0 && distance_on_read <= 10)
```

Comment:
Nothing about this PR but:
Where does this 10 come from. Is this a parameter which is normally 10 or do we want to give the opportunity to change it?

Reply:
This 10 is a totally arbitrary cutoff and in the end the user should get the opportunity to change it. When working on this PR I decided to leave this for later.
Shall I add a TODO or would you like to add a command-line option with this PR?

Reply:
No a TODO and or an extra issue is good. 👍

Reply:
Great, I added a TODO with 4ebeccee6e1ce4351e9c8699500b1c2d1b4fa124.